std/haxe/unit/TestRunner.hx

```diff
@@ -22,14 +22,39 @@
 package haxe.unit;
 import Reflect;
 
+/**
+	This class runs unit test cases and prints the result.
+	
+	```haxe
+	var r = new haxe.unit.TestRunner();
+	r.add(new MyTestCase());
+	// add other TestCases here
+	
+	// finally, run the tests
+	r.run();
+	```
+	
+	@see <http://haxe.org/manual/std-unit-testing.html>
+**/
 class TestRunner {
+	/**
+		The unit test results. Available after the `run()` is called.
+	**/
 	public var result(default, null) : TestResult;
+
 	var cases  : List<TestCase>;
 
 #if flash
 	static var tf : flash.text.TextField = null;
 #end
 
+	/**
+		Prints the given object/value.
+		
+		 * Flash outputs the result in a new `TextField` on stage.
+		 * JavaScript outputs the result using `console.log`.
+		 * Other targets use native `print` to output the result.
+	**/
 	public static dynamic function print( v : Dynamic ) untyped {
```

Comment:
I think it's worth noting that this function is `dynamic` so it can be overriden in custom setups.